packages/eslint-config-jest/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * strvcom/eslint-config-jest
+ *
+ * @author      Michal Jarnot <m.jarnot@yahoo.com>
+ * @copyright   2019 STRV
+ * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
+ */
+
+'use strict'
+
+module.exports = {
+  extends: ['plugin:jest-formatting/recommended'],
+  plugins: ['jest', 'jest-formatting'],
+  overrides: [{
+    files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
+    env: {
+      'jest/globals': true,
+    },
+  }],
+  rules: {
+    //
+    // Recommended
+    //
+
+    'jest/expect-expect': 'error',
+    'jest/no-disabled-tests': 'error',
+    'jest/no-focused-tests': 'error',
+    'jest/no-identical-title': 'error',
+    'jest/no-jasmine-globals': 'error',
+    'jest/no-jest-import': 'error',
+    'jest/no-mocks-import': 0,
+    'jest/no-test-callback': 'error',
+    'jest/no-test-prefixes': 'error',
+    'jest/valid-describe': 'error',
+    'jest/valid-expect-in-promise': 'error',
+    'jest/valid-expect': 'error',
+
+    //
+    // Optional
+    //
+
+    'jest/consistent-test-it': 'error',
+    'jest/lowercase-name': 'error',
```

Comment:
Ok, let's not bully people for name formatting. 😂 
```suggestion
    'jest/lowercase-name': 'off',
```

Reply:
Actually this is not bullying but teaching consistency and good practices. 😄 I am fine with both `lowercase` or `uppercase` but the test description should be consistent. A good example from HIL:

![image](https://user-images.githubusercontent.com/14946081/81109428-5d921d80-8f1a-11ea-9fee-de3098aaaf06.png)

Reply:
Actually it might make sense to use upper case when testing classes 🤔 I will remove the rule. ✅